### PR TITLE
Changed Scale Shot Effect Text

### PR DESCRIPTION
**Pull request by norman-codes**
### CONTENT
The effect text for Scale Shot (move ID 799) was modified to be more accurate and specific, in accordance with Issue #749.

**As part of doing so, 3 files were modified:**
1. _move_effects.csv_
The new effect_id 443 was added.

2. _moves.csv_
The effect_id of scale_shot was changed from 1 to 443.

3. _move_effect_prose.csv_
The new effect text under the ID 443 was added.

### TESTING

For testing, the instructions under "Setup" and "Database setup" were followed and the API was locally hosted.

Then, a GET request was made by accessing the Scale Shot move page in my browser:
`(localhost IP)/api/v2/move/scale-shot/`

### CODE REVIEW

File: data/v2/csv/move_effect_prose.csv [hunk 1]
```diff
@@ -1,124 +1,124 @@
...
-5,9,Has a $effect_chance% chance to [burn]{mechanic:burn} the target.,Inflicts [regular damage]{mechanic:regular-damage}.  Has a $effect_chance% chance to [burn]{mechanic:burn} the target.
-6,9,Has a $effect_chance% chance to [freeze]{mechanic:freeze} the target.,Inflicts [regular damage]{mechanic:regular-damage}.  Has a $effect_chance% chance to [freeze]{mechanic:freeze} the target.
-7,9,Has a $effect_chance% chance to [paralyze]{mechanic:paralysis} the target.,Inflicts [regular damage]{mechanic:regular-damage}.  Has a $effect_chance% chance to [paralyze]{mechanic:paralyze} the target.
-8,9,User faints.,"User [faint]{mechanic:faint}s, even if the attack [fail]{mechanic:fail}s or [miss]{mechanic:miss}es.  Inflicts [regular damage]{mechanic:regular-damage}."
-9,9,Only works on sleeping Pokémon.  Drains half the damage inflicted to heal the user.,[Fails]{mechanic:fail} if not used on a [sleep]{mechanic:sleep}ing Pokémon.  Inflicts [regular damage]{mechanic:regular-damage}.  [Drains]{mechanic:drain} half the damage inflicted to heal the user.
+move_effect_id,local_language_id,short_effect,effect,
```
Comment (Naramsim) [inline data/v2/csv/move_effect_prose.csv]: Hi, why did you add all the `,`?

Comment (norman-codes) [reply data/v2/csv/move_effect_prose.csv]: I think it's because I edited the .csv file in Microsoft Excel and it automatically did that. I'm sorry - I should have opened it in a text editor instead and only added that so that no formatting issues like that occurred. I'll update this PR and remove the unnecessary changes.

**Review by Naramsim — APPROVED** [review]

**Review by Naramsim — APPROVED** [review]

**Comment by Naramsim** [comment]
Thanks!

**Comment by pokeapi-machine-user** [comment]
A [PokeAPI/api-data](https://github.com/PokeAPI/api-data) refresh has started. In 45 minutes the staging branch of [PokeAPI/api-data](https://github.com/PokeAPI/api-data/tree/staging) will be pushed with the new generated data. <br><br> The staging branch will be deployed in our [staging environment](https://staging.pokeapi.co/api/v2/) and you will be able to review the entire API. <br><br> A Pull Request ([master](https://github.com/PokeAPI/api-data/tree/master)<-[staging](https://github.com/PokeAPI/api-data/tree/staging)) will be also created at [PokeAPI/api-data](https://github.com/PokeAPI/api-data/pulls) and assigned to the PokeAPI Core team to be reviewed. If approved and merged new data will soon be available worldwide at [pokeapi.co](https://pokeapi.co/api/v2/).

**Comment by pokeapi-machine-user** [comment]
The updater script has finished its job and has now opened a Pull Request towards [PokeAPI/api-data](https://github.com/PokeAPI/api-data/pulls) with the updated data. <br><br> You can see the Pull Request deployed at our [staging environment](https://staging.pokeapi.co/api/v2/) when [CircleCI deploy](https://app.circleci.com/pipelines/github/PokeAPI/deploy) will be finished (_check the started time of the last build_).
